frontend/src/components/v-summary-charts.vue

```diff
@@ -45,7 +45,7 @@
           span.tooltip-text Click to view group's repo
```

Comment:
The icon for this still shows a GitHub logo, might have to find another icon that is more suitable?

Reply:
😅 I can't think of anything too good. I think the `git` logo might be fitting for this though I'm unclear on anything related to licensing on this matter.

Reply:
Hmm the git logo seems weird here, what about `fa-file-code` or `fa-file-lines`? Gitlab seems to use the file-lines icon:

<img width="210" alt="Screenshot 2022-03-25 at 18 31 16" src="https://user-images.githubusercontent.com/27919917/160104252-645d1d70-407a-4fd7-ab17-0588178b030d.png">

Reply:
Between the two, I prefer file-code. I'll put forward this suggestion though 
![image](https://user-images.githubusercontent.com/65345505/160138963-3ef3b632-96ae-417c-8ea9-7fbeee6725c7.png)

Since I think it's a pretty common image for remote repositories, especially in images illustrating the forking workflow

Reply:
I've updated it to `fa-file-code`, do take a look. Thanks!

Reply:
@damithc Prof, do you have any thoughts about the icon to use to represent the link to the actual repository? Previously we were using the GitHub logo and it looks like this:
<img width="152" alt="Current" src="https://user-images.githubusercontent.com/27919917/160220230-c2bd7837-f42b-4757-934b-2fc148692862.png">

We are currently evaluating the following options to replace this logo so that it is more general and not GitHub-specific:
<img width="91" alt="Database" src="https://user-images.githubusercontent.com/27919917/160220137-42f093c1-eda2-433c-9b22-ad4d2a633ae3.png">
<img width="74" alt="File code" src="https://user-images.githubusercontent.com/27919917/160220145-e6c73942-ff3c-4047-8531-fe72a4cdcb2f.png">
<img width="81" alt="File lines" src="https://user-images.githubusercontent.com/27919917/160220153-896628a0-dd18-4e84-a042-b95c87114369.png">

Reply:
For generic icon, perhaps the `database` one? Also, fontawsome has this icon for git which may be suitable.
![image](https://user-images.githubusercontent.com/1673303/160223925-ca4d3756-870c-45e3-8d9c-40b03b683caf.png)

It would be cool if we can use respective github, gitlab, and bitbucket icons (fontawesome has all three) when we know the repo is one of them, but that can be a separate enhancement.

Reply:
I've chosen to settle on the `database` icon for now. I've tried the `git-alt` icon and it appears really small on the actual report with the branch lines being barely visible. 

This suggestion about the respective icons is really interesting! I think I'll create a new issue for it as I'm not too familiar with how to implement that right now.